article/_posts/2019-07-29-Dont-Worry-About-The-Weather.md

```diff
@@ -45,9 +45,9 @@ The main contributions:
 
 They select a reference sequence of images: daytime, clear, and overcast.
 Besides, they join some traversals with severe conditions: night, rain, snow, etc.
-Along with a cycle-consistency architecture, GAN to train generative models that can apply style transfer to the reference condition in order to create a number of synthetic sequences that maintain the structure and geometry of the reference condition.
+Along with a cycle-consistency GAN, that can apply style transfer to the reference condition creating synthetic sequences that maintain the structure and geometry of the reference condition.
```

Comment:
_Besides, they join some traversals with severe conditions (night, rain, snow, etc.), along with a (...)_

Reply:
Addressed in 9202b1c.